### PR TITLE
dx-531 created redirect CreditAccount

### PR DESCRIPTION
**Pull request by marinafrydenb**

### CODE REVIEW

File: payments/credit-account/redirect.md [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+                      icon="info"
+                      body="Swedbank Pay Credit Account is only available as a
+                      payment instrument for swedish merchants." %}
+
+## Introduction
+
+Redirect is the integration that lets Swedbank Pay handle the payments, while
+you handle your core activities. When ready to pay, the consumer will be
+redirected to a secure Swedbank Pay hosted site. Finally, the consumer will be
+redirected back to your website after the payment process.
```
Comment (asbjornu) [inline payments/credit-account/redirect.md]: Great introduction!

File: payments/credit-account/redirect.md [hunk 2]
```diff
@@ -0,0 +1,386 @@
+---
+title: Swedbank Pay Payments Credit Account Redirect
```
Comment (asbjornu) [inline payments/credit-account/redirect.md]:
```suggestion
title: Swedbank Pay Credit Payments – Redirect
```

File: payments/credit-account/redirect.md [hunk 3]
```diff
@@ -0,0 +1,386 @@
...
+      title: Redirect
+    - url: /payments/credit-account/after-payment
+      title: After Payment
+    - url: /payments/credit-account/other-features
+      title: Other Features
+---
+
+{% include alert-review-section.md %}
+
+{% include jumbotron.html body="Swedbank Pay Credit Account is an online payment
```
Comment (asbjornu) [inline payments/credit-account/redirect.md]:
```suggestion
{% include jumbotron.html body="Swedbank Pay Credit Payments is an online payment
```

File: payments/credit-account/redirect.md [hunk 4]
```diff
@@ -0,0 +1,386 @@
...
+  cancelled beforehand. Please note that both a successful and rejected payment
+  reach completion, in contrast to a cancelled payment.
+* When you detect that the payer reach your `completeUrl` , you need to do a
+  `GET` request to receive the state of the transaction, containing the
+  `paymentID` generated in the first step, to receive the state of the
+  transaction.
+
+This is how the payment window might look like:
+
+**Ha med eksempel av screenshot her**
```
Comment (asbjornu) [inline payments/credit-account/redirect.md]: Fix screenshot pl0x.

File: payments/credit-account/redirect.md [hunk 5]
```diff
@@ -0,0 +1,386 @@
...
+    "cancellations": { "id": "/psp/creditaccount/payments/{{ page.paymentId }}/cancellations" },
+    "payeeInfo" : { "id": "/psp/creditaccount/payments/{{ page.paymentId }}/payeeInfo" },
+    "urls" : { "id": "/psp/creditaccount/payments/{{ page.paymentId }}c/urls" },
+    "settings": { "id": "/psp/creditaccount/payments/{{ page.paymentId }}/settings" },
+    "approvedLegalAddress": { "id": "/psp/creditaccount/payments/<paymentId>/approvedlegaladdress" },
+    "maskedApprovedLegalAddress": { "id": "/psp/creditaccount/payments/<paymentId>/maskedapprovedlegaladdress" }
+  },
+  "operations": [
+    {
+      "href": "https://api.payex.com/psp/creditaccount/payments/{{ page.paymentId }}",
```
Comment (asbjornu) [inline payments/credit-account/redirect.md]:
```suggestion
      "href": "{{ page.apiUrl }}/psp/creditaccount/payments/{{ page.paymentId }}",
```

File: payments/credit-account/redirect.md [hunk 6]
```diff
@@ -0,0 +1,386 @@
...
+      "rel": "update-payment-abort",
+      "method": "PATCH"
+    },
+    {
+      "href": "https://api.payex.com/psp/creditaccount/payments/{{ page.paymentId }}/authorizations",
```
Comment (asbjornu) [inline payments/credit-account/redirect.md]:
```suggestion
      "href": "{{ page.apiUrl }}/psp/creditaccount/payments/{{ page.paymentId }}/authorizations",
```

File: payments/credit-account/redirect.md [hunk 7]
```diff
@@ -0,0 +1,386 @@
...
+      "rel": "create-authorize",
+      "method": "POST"
+    },
+    {
+      "href": "https://api.payex.com/psp/creditaccount/payments/{{ page.paymentId }}/approvedlegaladdress",
```
Comment (asbjornu) [inline payments/credit-account/redirect.md]:
```suggestion
      "href": "{{ page.apiUrl }}/psp/creditaccount/payments/{{ page.paymentId }}/approvedlegaladdress",
```

File: payments/credit-account/redirect.md [hunk 8]
```diff
@@ -0,0 +1,386 @@
...
+        }
+    ]
+}
+```
+
+If a `GET` method is used from `paymentUI` with a `paymentToken`, the following
+operations can be returned, depending on state of the payment and the last
+transaction.
+
+```http
```
Comment (asbjornu) [inline payments/credit-account/redirect.md]:
```suggestion
```json
```

File: payments/credit-account/redirect.md [hunk 9]
```diff
@@ -0,0 +1,386 @@
...
+HTTP/1.1 200 OK
```
Comment (asbjornu) [inline payments/credit-account/redirect.md]:
```suggestion
```

File: payments/credit-account/redirect.md [hunk 10]
```diff
@@ -0,0 +1,386 @@
...
+Content-Type: application/json
```
Comment (asbjornu) [inline payments/credit-account/redirect.md]:
```suggestion
```

File: payments/credit-account/redirect.md [hunk 11]
```diff
@@ -0,0 +1,386 @@
...
+
```
Comment (asbjornu) [inline payments/credit-account/redirect.md]:
```suggestion
```

File: payments/credit-account/redirect.md [hunk 12]
```diff
@@ -0,0 +1,386 @@
...
+{
+    "href": "https://example.com/cancelUrl",
+    "rel": "redirect-merchant-cancel",
```
Comment (asbjornu) [inline payments/credit-account/redirect.md]: This operation isn't very valuable to the merchant, as it is the URL (s)he sent in the initial request. Don't we have other operations we can enumerate?

File: payments/credit-account/redirect.md [hunk 13]
```diff
@@ -0,0 +1,386 @@
...
+    "method": "GET"
+}
+{
+    "href": "https://example.com/completeUrl",
+    "rel": "redirect-merchant-complete",
```
Comment (asbjornu) [inline payments/credit-account/redirect.md]: This operation isn't very valuable to the merchant.

File: payments/credit-account/redirect.md [hunk 14]
```diff
@@ -0,0 +1,386 @@
...
+| └─➔&nbsp;`href`          | `string`     | The target URI to perform the operation against.                                                                                                                                                 |
+| └─➔&nbsp;`rel`           | `string`     | The name of the relation the operation has to the current resource.                                                                                                                              |
+
+{% include iterator.html
+        prev_href="index"
+        prev_title="Back: Introduction"
+        next_href="after-payment"
+        next_title="Next: After Payment" %}
+
+[abort]: /payments/credit-account/after-payment#abort
```
Comment (asbjornu) [inline payments/credit-account/redirect.md]:
```suggestion
[abort]: /payments/credit/after-payment#abort
```

File: payments/credit-account/after-payment.md [hunk 15]
```diff
@@ -6,6 +6,8 @@ sidebar:
     items:
     - url: /payments/credit-account/
       title: Introduction
+    - url: /payments/credit-account/redirect
```
Comment (asbjornu) [inline payments/credit-account/after-payment.md]:
```suggestion
    - url: /payments/credit/redirect
```

File: payments/credit-account/redirect.md (hunk 4 above)
Comment (marinafrydenb) [reply payments/credit-account/redirect.md]: waiting for Are to complete his work and give me screenshots

File: payments/credit-account/redirect.md (hunk 13 above)
Comment (marinafrydenb) [reply payments/credit-account/redirect.md]: this is now deleted from this file

File: payments/credit-account/redirect.md (hunk 12 above)
Comment (marinafrydenb) [reply payments/credit-account/redirect.md]: this is now deleted from this file

**Comment by marinafrydenb** [comment]
Hi @asbjornu ! Ive rebased and therefore reorganized the redirect file into the "credit" folder. And deleted the old "credit-account" folder, which was empty. Now I am awaiting screeenshots from @arebra before I can merge this into dev.

**Pull request merged**